### ISSUE TITLE
[DOCUMENT]: GIBCT Section 116 - Excel document has color contrast issues that could be improved

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3758]
## Description

VA 508 office has identified the [Excel Download Data on All Schools](https://www.benefits.va.gov/GIBILL/docs/job_aids/ComparisonToolData.xlsx) has some areas that use color as the only way to convey intent. This is a violation of Success Criteria 1.4.1. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* Color contrast ratios are 4.5:1 or greater

## WCAG or Vendor Guidance (optional)

* [Contrast (Minimum): Understanding SC 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

## Screenshots or Trace Logs

<img width="513" alt="Screen Shot 2019-11-26 at 4 18 16 PM" src="https://user-images.githubusercontent.com/934879/69677765-498fe900-1069-11ea-913d-8c67ba233e82.png">

**Comment by desireeturner** [comment]
@1Copenut The maintenance of this file is managed by the EDU team, so we will make recommendations to them regarding this update.

**Comment by desireeturner** [comment]
@1Copenut The Education team has been informed of this issue.

**Comment by desireeturner** [comment]
@1Copenut The file has been updated by EDU, so I believe this is ready for a round of testing.

**Comment by 1Copenut** [comment]
@desireeturner Thank you! I've let the VA 508 office know this Excel doc is ready for retesting.

**Comment by 1Copenut** [comment]
@desireeturner Thank you! I've let the VA 508 office know this Excel doc is ready for retesting.

**Comment by 1Copenut** [comment]
@desireeturner Thank you! I've let the VA 508 office know this Excel doc is ready for retesting.

**Comment by 1Copenut** [comment]
@desireeturner This issue appears to be resolved. I'll move it to close.

**Comment by dneel-bah** [comment]
@1Copenut In your last comment you mentioned this would be closed; is there something new that we need to raise to EDU, or can I close it?

**Comment by 1Copenut** [comment]
@dneel-bah Thanks for the heads up. I thought I closed this item. Will do so now and remove it from the 508 issue list.
